src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/scalable/ScoreVariantAnnotations.java

```diff
@@ -539,12 +539,14 @@ private static void scoreVariantTypeAndSetElementsOfAllScores(final List<String>
                                                                   final List<Boolean> isVariantType,
                                                                   final VariantAnnotationsScorer variantTypeScorer,
                                                                   final List<Double> allScores) {
-        final File variantTypeAnnotationsFile = LabeledVariantAnnotationsData.subsetAnnotationsToTemporaryFile(annotationNames, allAnnotations, isVariantType);
-        final File variantTypeScoresFile = IOUtils.createTempFile("temp", ".scores.hdf5");
-        variantTypeScorer.score(variantTypeAnnotationsFile, variantTypeScoresFile); // TODO we do not fail until here in the case of mismatched annotation names; we could fail earlier
-        final double[] variantTypeScores = VariantAnnotationsScorer.readScores(variantTypeScoresFile);
-        final Iterator<Double> variantTypeScoresIterator = Arrays.stream(variantTypeScores).iterator();
-        IntStream.range(0, allScores.size()).filter(isVariantType::get).forEach(i -> allScores.set(i, variantTypeScoresIterator.next()));
+        if (isVariantType.stream().anyMatch(x -> x)) {
```

Comment:
I'm unfamiliar with this code but I'm assuming this will not have any `true`s if there aren't any instances of this variant type.